keyboards/handwired/3dp660/3dp660.json

```diff
@@ -0,0 +1,158 @@
+{

```

Comment:
This is for VIA?  

If so, it doesn't go into this repo, and should be removed.

Reply:
You are correct. This is for VIA. I placed this file here so I wouldn't misplace it in my filesystem. My apologies, I'll remove it.